client/layout/index.jsx

```diff
@@ -151,10 +153,13 @@ class Layout extends Component {
 				<AsyncLoad require="layout/guided-tours" placeholder={ null } />
 				{ ! isE2ETest() && <AsyncLoad require="layout/nps-survey-notice" placeholder={ null } /> }
 				{ config.isEnabled( 'keyboard-shortcuts' ) ? <KeyboardShortcutsMenu /> : null }
-				<MasterbarLoggedIn
-					section={ this.props.sectionGroup }
-					isCheckout={ this.props.sectionName === 'checkout' }
-				/>
+				{ 'jetpack-cloud' === this.props.application && <JetpackMasterbar /> }
```

Comment:
There are so many things in this layout, and now we're adding just another set of masterbars 😅 I think Jetpack Cloud should be using a different layout, because it's designed to be so different from Calypso one.

Reply:
I disagree. I think we should be using the same layout. Having a separate layout will cause us problems in the long run. Since if we start using calypso sections will be using one type of layout and jetpack-cloud would be using a different one.